modules/extensions-support/src/main/java/org/mule/module/extension/internal/capability/metadata/ImplementedTypeCapability.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal.capability.metadata;
+
+import static org.mule.util.Preconditions.checkArgument;
+import org.mule.extension.annotations.Extensible;
+
+/**
+ * A metadata capability which marks that an operation is an
+ * implementation of a type annotated with {@link Extensible}.
+ * <p/>
+ * The runtime consequences of this capabilities depend on the runtime.
```

Comment:
Can you rephrase or remove "The runtime consequences of this capabilities depend on the runtime."?